token/cli/src/main.rs

```diff
@@ -2747,6 +2832,43 @@ fn app<'a, 'b>(
                     )
                 )
         )
+        .subcommand(
+            SubCommand::with_name(CommandName::SetTransferHookProgram.into())
+                .about("Set the transfer hook program id for a token")
+                .arg(
+                    Arg::with_name("token")
+                        .validator(is_valid_pubkey)
+                        .value_name("TOKEN_MINT_ADDRESS")
+                        .takes_value(true)
+                        .required(true)
+                        .index(1)
+                        .help("The token address with an existing transfer hook"),
+                )
+                .arg(
+                    Arg::with_name("new_transfer_hook_program_id")
```

Comment:
Would just `--program-id` be bad? I don't see where it would collide, and the subcommand is already within the transfer hook scope

Reply:
It's entirely a code thing, since you have to pass this parameter unless `--disable` is passed, and not via `--new-transfer-hook-program-id`. We can split the difference and go `"new_program_id"`?

Reply:
Ahh I see. Yeah, I think shorter the better for CLI args; `new_program_id` is cool

Reply:
Done!